Dockerfile

```
@@ -4,7 +4,9 @@
 FROM mosaicml/pytorch:1.13.1_cu117-python3.10-ubuntu20.04
 
 ARG EXAMPLE
-COPY ${EXAMPLE}/requirements.txt requirements.txt
+COPY requirements.txt requirements.txt
+COPY examples/${EXAMPLE}/requirements.txt ${EXAMPLE}-requirements.txt
 
 RUN pip install --no-cache-dir -r requirements.txt && \
-    rm requirements.txt
+    pip install --no-cache-dir -r ${EXAMPLE}-requirements.txt && \
+    rm *.txt
```

```suggestion
    rm requirements.txt ${EXAMPLE}-requirements.txt
```
Forward compatibility in case we (or a user's fork) adds any other txt files